prow/kube/prowjob.go

```diff
@@ -63,4 +63,7 @@ const (
 	PullLabel = "prow.k8s.io/refs.pull"
 	// RetestLabel exposes if the job was created by a re-test request.
 	RetestLabel = "prow.k8s.io/retest"
+	// OptionalLabel is added in resources created by prow and
+	// carries the Optional from a Presubmit job.
+	OptionalLabel = "prow.k8s.io/optional"
```

Comment:
nit, `optional` might be useful for something more general later, can we do `is-optional` instead?

Reply:
done